src/lfortran/parser/parser.cpp

```diff
@@ -264,6 +264,16 @@ enum LineType {
     ContinuationTab, StatementTab, Include,
 };
 
+bool str_compare(const unsigned char *pos, std::string s) {
+    size_t i;
+    for (i = 0; i < s.size() && pos[i] != '\0'; i++) {
+        if (pos[i] != s[i]) {
+            return false;
+        }
+    }
+    return (i == s.size());
```

Comment:
I don't like using the loop variable after the loop, I think this is better:
```suggestion
    return true;
```